.github/workflows/run-smoke-tests.yml

```diff
@@ -0,0 +1,25 @@
+name: run-smoke-tests
+
+on: [push, pull_request]
+
+jobs:
+  test-smoke:
+
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        node-version: [12.x]
+
+    steps:
+    - uses: supercharge/mongodb-github-action@1.6.0
```

Comment:
> supercharge/mongodb-github-action@1.6.0 is not allowed to be used in apiaryio/dredd.

We're not going to be able to use this, perhaps it is worth removing this file/smoke tests from CI. Arguably merging this would still be better than current situation without smoke tests.

Reply:
Not a problem. Updated to use GitHub's services, I was lazy and copy pasting from an older project ;)

Passing build in https://github.com/dblock/dredd/actions/runs/9043948597/job/24852077761.